app/src/main/java/com/codesoom/assignment/filters/AuthenticationErrorFilter.java

```diff
@@ -0,0 +1,27 @@
+package com.codesoom.assignment.filters;
+
+import com.codesoom.assignment.errors.InvalidTokenException;
+import org.springframework.http.HttpStatus;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpFilter;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class AuthenticationErrorFilter extends HttpFilter{
+    @Override
+    public void doFilter(
+                HttpServletRequest request
+                , HttpServletResponse response
+                , FilterChain chain) throws IOException, ServletException {
```

Comment:
```suggestion
                HttpServletRequest request, 
                HttpServletResponse response, 
                FilterChain chain) throws IOException, ServletException {
```

일반적으로 comma를 뒤에 붙이고 아래로 내리는 것 같아요. 심지어 Golang은 지금처럼 comma를 작성하면 컴파일도 안돼요

![image](https://github.com/CodeSoom/spring-week7-assignment-1/assets/14071105/7d24fef5-0d8a-4e2e-aa65-9fa05208a776)